### ISSUE TITLE
No way to specify `python_gapic_opt` flag with `py_gapic_library` rule

### ISSUE THREAD

**Issue by aohren** [googleapis/gapic-generator-python#390]
We need to replicate the following protoc invocation from Bazel rule `py_gapic_library`. In particular, there's no current way to set `--python_gapic_opt` to the desired value below:

```
  protoc google/ads/googleads/${VERSION}/**/*.proto \
    --proto_path="${COMMON_PROTOS_PATH}" \
    --proto_path=. \
    --python_gapic_out=${OUTPUT_DIR} \
    --plugin=${GAPIC_GENERATOR_PATH}/gapic/cli/generate.py=python_gapic \
    --python_gapic_opt="old-naming,lazy-import,python-gapic-name=${GAPIC_NAME},python-gapic-templates=${GAPIC_GENERATOR_PATH}/gapic/ads-template"
```

where one var is custom:

`GAPIC_NAME` is `googleads`.

And the other vars are what you'd expect:

`COMMON_PROTOS_PATH` is api-common-protos dependencies
`GAPIC_GENERATOR_PATH` is root of gapic-generator-python

**Comment by aohren** [comment]
Output from a `bazel build --verbose_failures` run:

https://drive.google.com/a/google.com/file/d/1oKBkNbG6_wjwlNvS8F34CHQFBmLIpM8N/view?usp=sharing

**Comment by BenRKarl** [comment]
@aohren there's an updated value for `python-gapic-templates` now, it should point to the `/gapic/ads-templates` directory on the master branch of this repository.

**Comment by aohren** [comment]
@BenRKarl thanks for that. I've edited the original issue in-place to reflect the updated protoc arguments.

**Comment by vam-google** [comment]
Should be fixed in #411. Closing.

**Comment by software-dov** [comment]
Should be fixed by #402
